org.eclipse.scanning.api/src/org/eclipse/scanning/api/scan/ScanEstimator.java

```diff
@@ -97,35 +110,46 @@ public ScanEstimator(IPointGeneratorService pservice, ScanRequest<?> request, lo
 	 * @throws GeneratorException
 	 */
 	public ScanEstimator(IPointGenerator<?> gen, Map<String, Object> detectors, long timePerPoint) throws GeneratorException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)